Dockerfile

```diff
@@ -8,14 +8,6 @@ ENV PATH="/usr/src/app/bin:${PATH}:/opt/google-cloud-sdk/bin"
 # add our user and group first to make sure their IDs get assigned consistently
 RUN groupadd -r freight && useradd -r -m -g freight freight
 
-RUN mkdir -p /usr/src/app
```

Comment:
I believe this is still required, from the python:3.8.12 image:

```
root@4d5c49d5aa48:/usr/src# file /usr/src/app
/usr/src/app: cannot open `/usr/src/app' (No such file or directory)
```

Reply:
WORKDIR automatically mkdirs.